src/redux/categories/categories.js

```diff
@@ -0,0 +1,27 @@
+// action types
+
+export const SHOW_STATUS = 'categories/categories/SHOW_STATUS';
+
+// action creators
+
+export const showStatus = () => (dispatch) => {
+  dispatch({
+    type: SHOW_STATUS,
+    payload: ['under construction'],
+  });
+};
+
+// reducer
+
+const initialState = [];
+
+const categoriesReducer = (state = initialState, action) => {
+  switch (action.type) {
+    case SHOW_STATUS:
+      return [...state, action.payload];
+    default:
```

Comment:
- [ ] Please replace line 21 with

`return action.payload;`

because your state is dealing with a string :+1:

Reply:
resolved